src/components/PostDetails.tsx

```diff
@@ -146,7 +110,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </>
         )}
 
-        {loaded && !hasError && !visible && (
+        {loading && !err && !visible && (
```

Comment:
The condition `loading && !err && !visible` might not be correct for displaying the 'Write a comment' button. Typically, you would want to display this button after loading is complete, i.e., `!loading && !err && !visible`.